build.rs

```diff
@@ -12,4 +12,7 @@ fn main() {
         .args(["test/piggie.c", "-o", "test/piggie"])
         .status()
         .unwrap();
+    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
> If the build script inherently does not need to re-run under any circumstance, then emitting cargo:rerun-if-changed=build.rs is a simple way to prevent it from being re-run (otherwise, the default if no rerun-if instructions are emitted is to scan the entire package directory for changes). Cargo automatically handles whether or not the script itself needs to be recompiled, and of course the script will be re-run after it has been recompiled. Otherwise, specifying build.rs is redundant and unnecessary.

From the [documentation](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed) above it looks like `cargo:rerun-if-changed=build.rs` might be redundant.

Reply:
Removed.